.github/workflows/db_rollback.yml

```diff
@@ -0,0 +1,68 @@
+name: Rollback DB
+
+on:
+  workflow_dispatch:
+    inputs:
+      rollbackCount:
+        description: 'Number of migrations to rollback'
+        required: true
+        default: # of rollback you require in the file input
+      rollbackFile:
+        description: "File with migrations you want to rollback"
+        required: true
+        default: Filepath to yml
+
+jobs:
+  terraform-deploy:
+    name: Terraform Deploy
+    environment:
+      name: ${{ inputs.ENVIRONMENT }}
```

Comment:
Here's a `inputs.`, so that doesn't work in this instance.  This will need to convert to get the environment name from the above workflow dispatch input.